core/src/main/scala/org/apache/spark/ui/jobs/JobsTab.scala

```diff
@@ -29,4 +31,18 @@ private[ui] class JobsTab(parent: SparkUI) extends SparkUITab(parent, "jobs") {
 
   attachPage(new AllJobsPage(this))
   attachPage(new JobPage(this))
+
+  def handleKillRequest(request: HttpServletRequest) =  {
```

Comment:
Nit: extra space before brace, and extra parens around `killEnabled`.
Also I slightly wonder about `Option(request.getParameter("id")).getOrElse("-1").toInt` -- I doubt there would ever be a negative job ID but seems avoidable by just checking whether the param exists rather than filling in -1 for missing params. Still I don't think this does any harm at the moment.